tests/playwright/pages/editor-page.ts

```diff
@@ -904,6 +861,58 @@ export default class EditorPage extends BasePage {
 		await this.page.locator( EditorSelectors.panels.navigator.closeButton ).click();
 	}
 
+	/**
+	 * Set WordPress page template.
+	 *
+	 * @param {string} template - The page template to set.
+	 *
+	 * @return {Promise<void>}
+	 */
+	async setPageTemplate( template: string ) {
+		await this.openPageSettingsPanel();
+		await this.setSelectControlValue( 'template', template );
+		await this.waitForPreviewToLoad();
+	}
+
+	/**
+	 * Use the Default page template.
+	 *
+	 * @return {Promise<void>}
+	 */
+	async useDefaultTemplate() {
+		if ( await this.getPreviewFrame().$( '.elementor-default' ) ) {
+			return;
+		}
+
+		await this.setPageTemplate( 'default' );
+	}
+
+	/**
+	 * Use the Canvas page template.
+	 *
+	 * @return {Promise<void>}
+	 */
+	async useCanvasTemplate() {
+		if ( await this.getPreviewFrame().$( '.elementor-template-canvas' ) ) {
+			return;
+		}
+
+		await this.setPageTemplate( 'elementor_canvas' );
```

Comment:
The original function does this:
```typescript
		await this.getPreviewFrame().waitForSelector( '.elementor-template-canvas' );
```
WDYT?

Reply:
The selector is too specific. My intention to use generic methods (and selector).

The new `setPageTemplate()` method uses a generic check `await this.waitForPreviewToLoad()`.